### DISCUSSION TITLE
Which is faster Hono.js or Elysia.JS

### DISCUSSION THREAD

**Discussion by MohsinDev369** [honojs/hono#2310] [Polls]
I did some research but nothing useful, But found this on the ElysiaJS website. https://elysiajs.com/at-glance.html#performance hono is in third place. Is this true.

**Comment by yusukebe** [comment]
Elysia is faster on Bun. I'm not sure about the edge environment.

Benchmark results may vary depending on the environment and conditions. Therefore, it is not suitable for a questionnaire. If you do, please do it elsewhere.
